Src/LinearSolvers/MLMG/AMReX_MLMG.cpp

```diff
@@ -890,6 +900,9 @@ MLMG::actualBottomSolve ()
         else
         {
             MLCGSolver cg_solver(this, linop);
+
+	    if (nghost > 1) cg_solver.setNGhost(nghost);
```

Comment:
Why not just use `linop.getNGrow()` inside MLCGSolver?

Reply:
Because the original implementation does not use the same number of ghost cells everywhere. In the original implementation:
```
    const int nghost = sol.nGrow(), ncomp = sol.nComp(); //line 84
    ...
    MultiFab ph(ba, dm, ncomp, nghost, MFInfo(), factory); // line 90
    ...
    MultiFab sorig(ba, dm, ncomp, 0, MFInfo(), factory); // line 95```

I did not want to break anything by setting Multifabs `sorig,p,r,w,rh,v,t` to have `nghost=0`. However as long as it won't break anything I would be fine with implementing it that way.